uamqp/constants.py

```diff
@@ -170,3 +170,21 @@ class TransportType(Enum):
     """
     Amqp = 1
     AmqpOverWebsocket = 2
+
+
+class MessageBodyType(Enum):
```

Comment:
The spec refers to this as "message format" - so maybe we should call it `MessageBodyFormat`?

Reply:
I believe the term message format is reserved for the whole structure of a message, not just the body.

> 3.2.16 MESSAGE-FORMAT
MESSAGE-FORMAT: the format + revision for the messages fined  by this document 0
This value goes into the message-format field of the transfer frame when transferring messages of the
format defined herein.

> The exact structure of a message, together with its encoding, is defined by the message format. This
document defines the structure and semantics of message format 0 (MESSAGE-FORMAT). Altogether
a message consists of the following sections

e.g. the message is composed of header, delivery-annotations, properties, body, etc.

so we should keep the name type..